src/main/resources/i18n/Resources_zh_Hans.properties

```diff
@@ -33,12 +31,9 @@ preferencesUploaderStrategyPreferDuplicateSupportLabel=有部分要上传的文
 preferencesUploaderStrategyPreferDuplicateSupportDescription=之前已经用其他软件或是网页上传过的媒体文档会放入指定影集中，并以文件名称排序，或之后手动排序。
 preferencesUploaderStrategyPreferOrderLabel=无论文档是否已经用其他软件或是网页上传过，在影集中以媒体建立日期自动排序。
 preferencesUploaderStrategyPreferOrderDescription=上传的媒体在影集中会以媒体建立日期自动排序，但之前已经用其他软件或是网页上传过的媒体文档不会放入指定的影集中，上传完成后会在相关文档旁显示错误，需要手动放入影集中。
-#TODO translate
-preferencesDialogRelevantDirDepthTitle=How many levels of folders to use as albums
-#TODO translate
-preferencesRelevantDirDepthTitleFullLabel=Use all folders as albums, regardless of depth
-#TODO translate
-preferencesRelevantDirDepthTitleLimitedLabel=Use only this number of folder levels as albums:
+preferencesDialogRelevantDirDepthTitle=创建相册需要的子文件夹深度
+preferencesRelevantDirDepthTitleFullLabel=使用全部文件夹及其子文件夹创建相册
+preferencesRelevantDirDepthTitleLimitedLabel=仅使用指定子文件夹深度创建相册
```

Comment:
Is this one missing a colon (:) at the end? It'll be used in front of a text field with a number like this: 
`Use only this number of folder levels as albums: 2`

Reply:
Does the depth include the root folder itself?
so depth 2 means creating the album with files from Root folder and Folder1 only?
Root Folder -> Folder 1 -> Folder2

Reply:
depth 1 means disregard any sub-folders of the root folder, i.e. do not create any albums (for a programmer, it could be more natural to use a zero here, but for a user I thought it's best to use 1)
depth 2 means only use one level of sub-folders, i.e Root Folder -> Folder 1 

You are right that it's not really clear. I've changed the English version of this label to be: `Use only this number of folder levels as albums ('1' means do not use sub-folders as albums at all):`

Reply:
done. updated